packages/sdk/src/endpoints/Endpoint.ts

```diff
@@ -47,7 +47,10 @@ export abstract class Endpoint {
 
         if (httpResponse.status !== expectedStatus) {
             const errorPayload = httpResponse.data.error;
-
+            if (!errorPayload) {
+                const fullUrl = `${httpResponse.request.protocol}//${httpResponse.request.getHeader("host")}${httpResponse.request.path}`;
```

Comment:
I'd leave out protocol, host and only inject the path

Reply:
As this is mostly logged when connecting to the wrong server/port, I would leave it in there.

Reply:
as this is user specified information, please leave it out.

Reply:
The more I think about it I highly doubt we even need the path. The connector always responds with an error object (even for 404s).

Reply:
If you keep the host use `.request.host`, instead of retrieving it from the request header.